reference_validator/YAML_HotClasses.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python
+#coding=utf-8
+
+# File: YAML_HotClasses.py
+# Brief: Additional classes used for HOT reference validation
+# Classes: YAML_Env, YAML_Prop_Par, YAML_Reference, YAML_Resource
+# Author: Katerina Pilatova (kpilatov)
+# Date: 2016
+
+from __future__ import with_statement, print_function
+
+import YAML_Enums as ENUM
+import YAML_Hotfile as HOT
+
+class YAML_Env:
```

Comment:
Class names in Python should be CamelCase (see PEP-8 which OpenStack follows). So this should be called `YAMLEnv` and so on. But I think the yaml prefix is not really adding much.